### ISSUE TITLE
'CollectionQueryType' is not exported from  './use-swr-collection'

### ISSUE THREAD

**Issue by ChrisEbert** [lemasc/swr-firestore#1]
Hey!

First of all - thanks for your v9 port of swr-firestore!

I get the following error in my next.js typescript setup when importing the `useCollection` hook from `swr-firestore-v9`.

```
error - ../node_modules/swr-firestore-v9/lib/module/hooks/index.js
Attempted import error: 'CollectionQueryType' is not exported from './use-swr-collection'.
```

I'm not too familiar with the correct typescript setup in npm packages but mustn't types like `CollectionQueryType` included in the `lib/module` folder?

Or is there something wrong with my project configuration? 

Everything works fine when importing the files from `swr-firestore-v9/lib/commonjs` but I'm loosing the types when doing so.

**Comment by matigumma** [comment]
me too! 

Failed to compile.

./node_modules/swr-firestore-v9/lib/module/hooks/index.js
Attempted import error: 'CollectionQueryType' is not exported from './use-swr-collection'.

im using:

"react": "^17.0.2",
"typescript": "^4.1.2"

**Comment by matigumma** [comment]
in
**./node_modules/swr-firestore-v9/lib/module/hooks/index.js**
leave the line 2 as:
**export { getDocs, useCollection } from './use-swr-collection';**

**Comment by lemasc** [comment]
Sorry for late reply! I will fix this and publish a new version in a few days.

**Comment by lemasc** [comment]
Fixed in [v1.0.5](https://github.com/lemasc/swr-firestore/releases/tag/v1.0.5).
